### DISCUSSION TITLE
Missing xcconfig?

### DISCUSSION THREAD

**Discussion by SuzGupta** [mikaelacaron/Basic-Car-Maintenance#56] [General & Questions] [answered]
I just forked and cloned and cannot build because Basic-Car-Maintenance.xcconfig shows in red (reference is there but file is missing is what I think this means?). Should I just delete it? If not, what's the correct approach so I can build the project? Thanks in advance!

**Comment by mikaelacaron** [comment, marked as answer]
From the [README.md](https://github.com/mikaelacaron/Basic-Car-Maintenance#getting-started)
Close the project, and run this in Terminal `cp Basic-Car-Maintenance.xcconfig.template Basic-Car-Maintenance.xcconfig` then open it again and see if it's still red

**Reply by SuzGupta** [reply]
Thank you, sorry for the noise! 🤦‍♀️

**Reply by mikaelacaron** [reply]
All good! If you have any suggestions to make the README easier to read so that it's easier to follow the directions, let me know!
